src/modules/events/send_event.cpp

```diff
@@ -252,10 +252,14 @@ int SendEvent::custom_command(int argc, char *argv[])
 			return -1;
 		}
 
-		bool gyro_calib = false, accel_calib = false, baro_calib = false;
-		bool calib_all = true;
+		bool accel_calib = false;
+		bool gyro_calib  = false;
+		bool baro_calib  = false;
+		bool calib_all   = true;
+
 		int myoptind = 1;
```

Comment:
I know this seems line I am creating extra lines, but the reason for extra lines is that extra lines are free and the intent is explicit! :)

Reply:
Well, according to that logic I could add 50 newlines. I agree that extra lines can help with readability when placed carefully (and the changes here are good), but they're also not free as editor space is limited.
Generally please try to avoid/minimize whitespace changes only, because they make it harder to track something in the git history and lead to more rebase/merge conflicts.

Reply:
Agreed.  I'll revert these.  As always, thanks for your feedback!  I appreciate it!